src/PactNet.Abstractions/Verifier/IPactVerifierSource.cs

```diff
@@ -7,13 +7,14 @@ namespace PactNet.Verifier
     /// Configured pact verifier source
     /// </summary>
     public interface IPactVerifierSource
-    {
+    {

```

Comment:
Are there some line ending changes here or something? Unchanged lines appear to be marked as change